gui/autofish.lua

```diff
@@ -0,0 +1,110 @@
+-- config ui for autofish
+
+local gui = require("gui")
+local widgets = require("gui.widgets")
+local script = reqscript("autofish")
+
+local REFRESH_MS = 10000
```

Comment:
in order for autorefresh to be useful, you should display some live statistics on your window, such as how many raw and processed fish there currently are. Similarly, adding a minimal mode would allow the players to use your gui as a small status window that they can leave up on the screen.

Reply:
It does currently show whether or not autofish has disabled/enabled fishing, and this can also be useful if settings were updated outside of the GUI for some reason. I don't see a reason to *not* add a fish counter to it though, to make it more useful.